db.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import json
+
+def db_to_workstreams_md(db):
+    output = """# WHATWG Workstreams
```

Comment:
It's not clear where this document would be deployed. If it's to the website, then HTML seems potentially better as an output.

Reply:
This is a sample to generate the existing Workstreams.md resource. One way to go would be that initially we don't really change anything, except for where the data originates and then we change/obsolete some outputs, but also happy to merge those. (This will likely take a while anyway as I don't expect everything to be ready and reviewed by this Friday and after that I'll be out for some time.)

Reply:
IMO the existing Workstreams.md resource is best replaced. It has two purposes:

1. The source of truth, which the SG controls, and which our agreements can point to.
2. An externally-readable summary for people wondering what the WHATWG workstreams are.

(1) should be replaced by JSON. (2) should be on the website in HTML, I think, but the overlap with https://spec.whatwg.org/ makes it a bit unclear what the right path would be there.

Reply:
That's fair, in that case I suspect we want a separate document for Workstreams that's similar to this Markdown resource as they are primarily an IPR device. Keeping the spec namespace focused on standards only seems preferable to me.

Reply:
Yeah, that's probably the easiest route. You could imagine something clever that adds little collapsed workstream annotations or something to the side of spec.whatwg.org, or some other merger, but it'd require too much design thinking probably.